lib/Slackbot_worker.js

```diff
@@ -1,17 +1,21 @@
 var Ws = require('ws');
+var clone = require('lodash.clone');
```

Comment:
Can you use `lodash.clone()` function? Instead of adding a library only serves as a `clone` creator.